### DISCUSSION TITLE
How to adjust the update interval?

### DISCUSSION THREAD

**Discussion by redpixy** [tony-fav/tasmota-blerry#54] [Q&A]
Hi, is there a way to change the update interval of the devices connected to Blerry?

**Comment by tony-fav** [comment]
Can you clarify what you are wanting to do?

Blerry sends updates states over mqtt as it receives updated states over Bluetooth advertisements. It doesn't delay them by more than a couple seconds (it has a queue so it only sends a subset each cycle).
